src/Exception/ExceptionWithFrontendLabelTrait.php

```diff
@@ -1,38 +0,0 @@
-<?php declare(strict_types=1);
-
-namespace Becklyn\RadBundle\Exception;
-
-use Becklyn\RadBundle\Translation\DeferredTranslation;
-use Symfony\Component\Form\Exception\UnexpectedTypeException;
-
-/**
- * @final
- */
-trait ExceptionWithFrontendLabelTrait
```

Comment:
This is technically a BC break, but this class was used nowhere so far, as it just recently was added.